service.go

```diff
@@ -254,11 +189,65 @@ func (svc *Service) StartSubscription(ctx context.Context, sub *nostr.Subscripti
 	}
 }
 
-func (svc *Service) HandleEvent(ctx context.Context, event *nostr.Event) (result *nostr.Event, err error) {
-	//don't process historical events
-	if !svc.ReceivedEOS {
-		return nil, nil
+func (svc *Service) PublishEvent(ctx context.Context, sub *nostr.Subscription, event *nostr.Event, resp *nostr.Event) {
+	status, err := sub.Relay.Publish(ctx, *resp)
+	if err != nil {
+		svc.Logger.WithFields(logrus.Fields{
+			"eventId":      event.ID,
+			"status":       status,
+			"replyEventId": resp.ID,
+		}).Errorf("Failed to publish reply: %v", err)
+		return
 	}
+
+	nostrEvent := NostrEvent{}
+	result := svc.db.Where("nostr_id = ?", event.ID).First(&nostrEvent)
+	if result.Error != nil {
+		svc.Logger.WithFields(logrus.Fields{
+			"eventId":      event.ID,
+			"status":       status,
+			"replyEventId": resp.ID,
+		}).Error(result.Error)
+		return
+	}
+	nostrEvent.ReplyId = resp.ID
```

Comment:
Can we address https://github.com/getAlby/nostr-wallet-connect/issues/231 in this PR?